Makefile

```diff
@@ -77,6 +77,10 @@ check: setup-ckb-test ## Runs all of the compiler's checks.
 build: ## Build binary with release profile.
 	cargo build ${VERBOSE} --release
 
+.PHONY: build
```

Comment:
may  typo, should be "build-for-profiling"

Reply:
Yes, it's a typo. I will force-push this PR to fix it.